the-faseb-journal.csl

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with=", " names-delimiter=", " page-range-format="expanded" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
could you remove the intialize-with and the names-delimiter here? I don't think you want that and it will make it harder to work with the style for others.